README.md

```diff
@@ -17,11 +17,10 @@ analysis in order to understand your software's performance and behavior. You
 can export and analyze them using [Prometheus], [Jaeger], and other
 observability tools.
 
-*Compiler support: [requires `rustc` 1.70+][msrv]*
+*[Supported Rust Versions](#supported-rust-versions)*
```

Comment:
This [section](https://github.com/open-telemetry/opentelemetry-rust?tab=readme-ov-file#supported-rust-versions) needs some changes, the MSRV varies across crates—some require Rust 1.70, while others need 1.71.